libs/api/transaction/feature/src/lib/api-transaction-feature.controller.ts

```diff
@@ -39,8 +40,8 @@ export class ApiTransactionFeatureController {
   @ApiBody({ type: MakeTransferRequest })
   @ApiOperation({ operationId: 'makeTransfer' })
   @ApiResponse({ type: AppTransaction })
-  makeTransfer(@Body() body: MakeTransferRequest) {
-    return this.service.makeTransfer(body)
+  makeTransfer(@Body() body: MakeTransferRequest, @Req() req: Request) {
+    return this.service.makeTransfer(body, req)
```

Comment:
Let's swap the order of these parameters as discussed yesterday: 

- `@Body() body: MakeTransferRequest, @Req() req: Request` => `@Req() req: Request, @Body() body: MakeTransferRequest`
- `(body, req)` -> `(req, body)`